ui-v2/app/templates/dc/services/show.hbs

```diff
@@ -53,7 +53,7 @@
                     href=(href-to 'dc.nodes.show' item.Node.Node)
                     data-test-node=item.Node.Node
                     name=item.Node.Node
-                    address=item.Node.Address
+                    address=(concat item.Node.Address ':' item.Service.Port)
```

Comment:
Would it make sense to add `port` as an argument to `healthchecked-resource`?

Reply:
😄 Yeah I did consider that and backed out, I figured that an 'address' is both the IP and the port joined together, kind of like an 'address' is a house number (port) and street name (ip).

Reply:
Cool! My only thought was it would be easier to change the styling/etc of the port in the component. Cheer!

Reply:
Yeah for sure, if I was needing to do that I'd probably invert the control go with the slot/block route. Right now I've been let off :smile: